conda-recipes/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set VERSION_SUFFIX = "" %} # debug version suffix, appended to the version
+
+package:
+  name: numba_rvsdg
+  # GIT_DESCRIBE_TAG may not be set
+  version: {{ "%s%s" % (environ.get('GIT_DESCRIBE_TAG', '').lstrip('v'), VERSION_SUFFIX) }}
+
+source:
+  # Using the local source tree helps test building without pushing changes
+  path: ../
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python >=3.11
+    - python-graphviz
+    - pyyaml
+
+test:
+  requires:
+    - pytest
+  commands:
+    - pytest --pyargs numba_rvsdg
+
+about:
+  home: https://github.com/numba/numba_rvsdg
+  license: New BSD License
```

Comment:
I believe that the license specification here is incorrect. According to the OSI the `New BSD License` is:

https://opensource.org/license/bsd-3-clause/

However, the license document in the `README.md` only has two clauses. 

This is also what GitHub has identified when analyzing this repository:

<img width="315" alt="Screen Shot 2023-08-08 at 11 09 41" src="https://github.com/numba/numba-rvsdg/assets/41563/2c8982de-01cf-4847-8d26-aee0755c5333">

So this is the 2-Clause BSD license as per the OSI:

https://opensource.org/license/bsd-2-clause/

And as such I would recommend to change the line as follows:

```suggestion
  license: Simplified BSD License
```